### DISCUSSION TITLE
Add an organism that assimilates humans.

### DISCUSSION THREAD

**Discussion by kineticcrusher** [FakeFishGames/Barotrauma#8008] [Ideas and suggestions]
Husk infection is caused by a parasite which hijacks the host body, but it does so in a very obvious form and functionally kills the player. An interesting addition to the Europan fauna could be an amorphous organism - perhaps found in alien ruins or derelict subs - that could infect and subsequently assimilate a human player, such that they appear identical to their normal selves and still have control of their character, but are no longer a human and must spread the infection to the rest of the crew - perhaps by, in a style comparable to that of The Thing, revealing themselves and violently attacking bystanders.

Since such a mechanic might conflict with the idea of traitors, it should be possible for server owners to enable or disable the spawning of the organism - either that, or it could be tied to difficulty levels.

As a game element, it would add a really unique form of distrust - dive teams could no longer be trusted to be themselves by return, shuttle crews could suspect submarine crews as being infected, crews could split up and throw around accusations, and ultimately it would help drive home the point that Europa is a highly dangerous place with no intention of supporting the human species.

**Comment by itchyOwl** [comment]
Good ideas. Maybe you can try your hand in modding?

> Husk infection is caused by a parasite which hijacks the host body, but it does so in a very obvious form and functionally kills the player.

Did you know that you can modify the husk infection (or a custom infection using the type "alieninfection") so that it doesn't kill the targets, for example like this, which prevents the transition to the final state of the infection:
![image](https://user-images.githubusercontent.com/5370738/148738685-05d5f85d-face-42ef-a7b6-551095debf87.png)

**Comment by IgRAzm** [comment]
I think a unique enough way to do The Thing-like infection could be as a mutated form of husk parasite that has more sentience. If this infection has gotten to a point, it can't be self-healed because it doesn't let you use the cure yourself; visible to a medic symptoms take longer to show, and visible to everyone effects take even longer, also it gives tasks like with traitors, however here they are usually about infesting crew members, supplies, and keeping yourself out of danger; failing to do the tasks can eventually kill you and turn into a normal husk-infected.
